src/app_service/service/message/async_tasks.nim

```diff
@@ -28,6 +28,7 @@ const asyncFetchChatMessagesTask: Task = proc(argEncoded: string) {.gcsafe, nimc
     var messagesArr: JsonNode
     var messagesCursor: JsonNode
     let msgsResponse = status_go.fetchMessages(arg.chatId, arg.msgCursor, arg.limit)
+    sleep(3000)
```

Comment:
Artificially delay async fetches.

Reply:
Why? This would introduce some slowness to the channel switching.

We want to see the animation, but not just for the sake of seeing the animation.
It feels better to switch, see an animation for 0.5 seconds, then having the message, instead of switching, freezing for 0.5 seconds and then seeing the messages.